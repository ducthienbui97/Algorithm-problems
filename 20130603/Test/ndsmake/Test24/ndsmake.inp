2447
154892665 944134321 -321798947 614191321 -174805681 617919182 -964913774 -822796837 980017058 924105439 -434051675 -4321920 578404021 -166139295 -320244972 -985068867 -42978741 -29194652 154712551 -680931647 763043006 128319543 -420032370 -366499162 -823600337 902592177 -615124889 -152538412 -237414697 -784350690 534758491 -820476429 -461695803 6143912 484575626 825285139 919658169 -280160711 763756052 -923328922 743527473 147074447 -756383223 -403281545 -210451450 -720112759 107556807 768316772 -790507830 -51363882 -821310375 723635974 -923011175 -110987557 -408040769 551005684 -15414372 -273447760 -691946726 331314539 -184281854 -274130321 -506368997 411453741 826371105 -46510688 598496905 -907859521 -722871767 -778963474 -414713370 691221503 -925804878 -137478982 216076328 872652468 -754063225 -460946485 -471471648 196912925 865376376 591101639 842146635 -36078432 217753530 -232172314 -740575037 -550328062 -231281108 397538266 -479946928 -162435678 -627529445 -568674602 -132995019 -404104023 588770854 -832807743 -785661017 327490254 -869848511 206170135 -392635239 502379462 353783508 833577565 -485042254 440699719 725643011 350166334 315662176 806589624 -288968951 699640647 572860166 112991095 346523581 -470390212 815083345 -862386746 347045943 -352446187 -696271448 447355987 19963788 -573140072 -628008767 401866850 683669684 -319132268 -860275631 -999220583 339265463 -281487908 -865702809 475729401 678254605 34929834 -822882798 283282466 920010773 -453675302 -463092899 380824194 108428688 -87576366 507211052 454085259 -13036983 -870851663 -717900322 366900975 901388112 -974438795 888191569 -64275831 -454697985 -716362920 -712977269 22542315 -597732809 -282672550 -224801845 683824714 370225099 -565024941 -737188855 -682873913 -218483612 -245396672 -652265960 400257482 177230764 -149330906 931290218 125316199 -702474575 -768420488 287765956 444537149 337376765 844342863 -891739973 -255842479 314719413 -432169676 211403771 304536634 81676701 751079074 -948020332 -658719636 693711138 -200471569 550728177 943379488 642579414 -652026817 347783115 484535957 -833088289 -19353482 -202995682 -52746521 -498130844 928629643 -281212978 242010255 -828351586 710897375 -4173207 -259730109 -301008291 538876342 104712045 398141495 -98631423 465476544 424996659 -415796999 -307365951 -84105566 872127497 -257732059 80406729 441200458 724921396 -683232326 -391038902 962402704 858201428 813977559 850897834 -110054264 -900722410 -864849444 223736166 775280866 -746144638 -566616373 935452430 -221057148 -20184778 397061105 -423619528 -7624813 -820727804 46617149 -628859570 -611567075 899030991 943278720 844856253 499982835 431688222 -145158020 39825093 -428875566 666002021 -247645284 176894787 -414318315 -383479171 -890654998 -556577624 -253330697 -265358496 -655739102 437306968 -528120192 237367427 -153458778 -320257136 -826771602 -979561593 -389569948 423126279 -696305784 -359971754 388909204 457910426 51616368 -302651216 535441498 706119220 54762665 423147372 410051551 -156390935 193743177 579354076 -7178226 465599778 -732622788 -202106546 955176168 -863743757 104597167 185657733 762551177 -739266778 -140834229 -303113537 -737986420 -161607249 61223677 -15217201 -200843241 466588080 -755267599 996235301 -69746044 -510456274 107052879 -876117306 -926335987 108672690 -90696175 -230819729 726389814 702313867 -231487850 -417249481 139706742 605404977 965234112 837755596 951133042 -873905793 419682720 -862606089 -953928769 935359056 -971072255 513963718 488931525 297676210 102681119 -704107940 399357119 -315408950 196888181 989382481 -339047714 -516078012 -187528780 -435749945 -390155123 -173101708 -192661040 740279306 -725970736 342736142 127612467 -476701981 47167430 -203238274 51192954 -620004275 175187685 -862605853 992599525 -301161020 -453828942 51103708 -314836143 -314850325 664300475 -333562165 109158119 120261041 530570639 -603377236 -582130333 -717202065 91466662 -426259809 452221648 -593967558 -585122905 409741904 672065484 -88991252 -889596225 777730227 -476901727 425902437 -825719052 -748122898 287991268 -235209845 -980338795 -607606300 -155067586 323818977 401255122 305543519 -743202745 632444099 -97520024 751275994 508543886 562292028 -935241355 -675519768 84809653 911921796 180575593 -468698045 259795641 -916481767 -450767843 597480309 -903598256 153314558 903878684 206953067 376966264 666073953 -911452450 621303515 535993156 -237795333 -171605270 -585330417 301702387 -310270104 -348570373 -173552698 804380649 567302650 -596764651 -153148419 -774354901 -285119815 -488502919 -256434602 -153777854 19391021 -31729629 -478392907 944214443 395288446 -714775802 -318943310 -200738158 -452578935 -271575476 -344926110 -566565123 754182102 539122660 386900567 100132073 196612859 -69730087 -797957331 -162315723 684511234 792732578 634196962 -720145527 489654711 422396627 -398506924 -76500039 -587953292 316796350 714211599 -609431699 806126834 -495230681 -360904725 876591009 -899945591 150023262 -988576007 757795839 -277344960 -22053905 327701974 645118797 -790826740 15846788 823444140 382418432 739585636 -644702089 -725172924 276869209 887217012 -709472229 -818426134 -818580504 379125634 -675139005 627014013 -207173308 -619879157 718039129 -442669026 695551085 -110939269 -659132530 -122830732 422718087 180682393 145290856 -985007829 641006599 -281188956 701166464 -805647813 447419409 429305555 501602967 680247464 -948347635 698958165 363423966 -802505390 68231004 964222130 -504961573 -272865148 332971507 -978894888 397229743 -718726895 797145198 80082627 498258696 -301859779 -296806201 -352552768 757741590 888510204 444555645 -977973433 -484711559 874183649 491302472 663696285 -51557475 608087631 -960853373 123512856 771220309 -669236345 -751889335 -337823021 -157100873 -1760288 -226391601 -320385132 706035650 63973173 -651098734 905430965 673529783 -640938520 -203910029 901340664 -301791980 514525878 921625712 679231841 -918094150 719587839 -248101181 697553846 -663190907 -428852156 -152283200 -791952146 220641190 -383632706 905757609 898557075 -904878689 -519555273 767918724 -53787098 -567603323 459915016 -153877968 774310585 -781132124 -560006479 901365223 -952259751 335103343 -590929677 985383801 -605902160 422790395 -849126517 92673271 -433567899 110533104 646808545 -85972706 976518469 -945037847 -710838454 -126976924 29369655 159612728 526101160 -754996572 303091514 565858383 -710738645 -123998476 166304278 624898848 916714073 735004647 444229838 870521469 169088240 158481537 724059838 531270647 927622009 778043216 629980405 994938348 -712350463 376862398 -695618996 -143324748 245243981 163275978 -118830216 -964605449 795977979 -384381626 -629148201 748763984 -521194289 -340820959 -436639909 -817334062 -775642009 405056590 298461383 661682139 862355508 226227017 -72724033 -128502814 838905661 423546190 167099587 224634052 227432691 307365948 -13734335 -655627534 269219493 -84012942 467597986 481811461 279563947 -835704406 895708161 719065834 45361437 607180441 -844239738 124557032 -297187304 384517158 209519616 892134673 -418830773 502300128 -895284431 -368245288 -853307800 100767000 -22183569 581956331 913907447 -237088482 94273744 668751843 -394185123 167371253 940012332 865186266 724848646 682641199 193784909 632575509 -865678802 -673709325 849063379 -319947187 -348865759 -481046873 -958597775 -597805539 -877163892 -809197542 -786269172 -229726391 -96213914 -765228391 973431593 -42385485 -235140166 -472794097 790258376 -195122078 134627602 371728855 793516261 83076359 426502200 599705692 597367306 266544682 289802266 -346269104 18692349 -255728650 -570513721 596384133 -685484135 30007197 975479701 -970699925 694694586 -335959703 -330837659 -244534489 -82592942 308377953 -475188481 896815948 734899865 295757972 -242787433 -878751432 498775301 98375025 -672566258 814137991 -988767694 -997683040 -194553525 340357253 909109315 -740590614 131602695 -488215003 -810015796 -812086841 925527632 548378687 -835124344 -134855671 603794581 802786620 236499813 -519619385 691034063 -509205859 -321913622 -887261059 -478826220 228917118 -639064648 923478930 -754918176 562432839 628276748 355860436 -731009008 321495820 -760615215 -870798461 -258186814 780853959 267961169 301415973 28031321 -803103736 -485543962 -721364323 998893908 36355221 481869225 912552447 -992999734 595442607 -910744838 179761120 177476834 503490786 -71615852 649617269 372475090 325151935 -183515133 104592728 391669432 98683359 -187081363 985021618 572055664 24260384 -992552248 279915491 405485638 -528207500 316982497 514980958 166369477 77240390 39307896 869033504 845873070 915113604 440351111 -737421106 705686020 -875042227 380257955 -612963222 494956356 -886964935 -694736772 929751985 -192111181 207694207 648591893 12782031 511637063 594735514 -279259310 993174247 444914765 97806879 127148719 860094151 -467122642 452967874 836423636 -824807797 -863851245 761206461 -270737598 151015060 -656595849 -854794989 -418255758 907486105 -321198378 386553584 493716366 -400869675 347587316 -734221009 -520169274 527139678 -860630074 557741558 -938583215 870442646 447909121 -141293700 630668801 390374842 355059262 473101547 -276648064 846822040 -10613049 577510444 -197691043 -650890239 -571038448 -577670259 -315080308 221545589 521369732 243796847 -667022325 -530908803 -351745531 385768097 717965400 -845672057 -50594542 390175448 -719796983 27018254 -233286432 90779216 -769089535 43088887 -684567087 -935433619 543664679 532529744 -652053334 -965525123 46032819 68110113 859100412 244636868 179698965 -495174699 470147801 284564852 938011073 281577432 757566943 -535260043 -266274812 -535932682 -333892975 47958631 770395785 -897152565 499850022 600774586 -279753623 -752837481 -67959743 182537019 142896622 -959504937 488801411 646967101 868645746 -782315905 -641000645 468547429 -746945433 25774039 -946915272 -423608047 -949663632 -613397777 -240996672 -722407323 326881451 876891584 561128857 -786607116 -441378030 -316112927 -177195452 891144 -837821961 501222729 379337359 -556176979 720675401 -167922526 -778879419 838971175 -755525247 702260914 66988918 -3419307 -251197290 83369040 -213238199 -448469861 603345605 397382531 -949254081 933007145 551019325 26803758 -770486445 287521463 -304680226 -871966272 152100643 -809153464 -548993477 108697724 -65384501 666542553 386731051 -742748191 837041365 305750251 864586684 567164798 -611626112 -113434545 -896612627 577072915 226448381 -744153737 -320785130 -67270216 -673844888 682686964 -502904548 262170721 -231447858 423527253 -399076629 893718219 811206441 -886237348 607029833 672813856 -637696731 -32376964 -854365688 -707163191 -183089242 170660852 -121002939 360055694 36800175 917625171 -909191891 634832299 357291329 4278896 452447173 935365174 -156566291 -948272275 -756507982 706602105 -286792597 156276545 -552871087 -64946786 767018494 132760434 83480547 16650231 793717238 246211951 781858534 432409832 792905551 -148198516 -169426561 -88770702 648711287 650025882 -262062318 -603114963 -534009566 -416305164 435353855 710859397 314160446 -259944136 842540463 -146579960 -585829867 -595274842 397242562 938281208 -217725967 -924446774 -535776962 561687216 -509126145 -792811224 -799166429 -104597206 262985890 -452152691 -694940847 -701404517 170430324 108758400 -716694249 397199161 130438362 293674712 -538537382 654955804 471974210 -121408921 623049055 -392519614 -999119876 -528873516 -133942475 701868978 993219206 -872001086 734470311 698145925 170546876 -730330008 583197990 552799686 676351418 -617441965 -484165046 811026766 -797500642 -911394462 -759405054 950568571 264537425 -841296721 -599270713 -756858087 685098429 -85171039 853465606 761663562 146778429 -650393123 -684097476 681712541 872728701 -100345216 939433752 218709913 789038189 -566864035 -343447279 65116333 -55007408 -657654910 507456095 123938556 -965987911 397880839 310207903 -340622899 171270186 -648684427 507609653 556138540 184355870 188146295 -982794918 -812137289 258947038 -701209149 627879778 -777505522 203979962 97783187 559221398 578479417 174031653 281621792 -19492935 -795698435 -786396247 937906207 -622373399 -893133740 793075163 -58550639 -358210084 -593864863 -359245550 -390842040 429531574 -172861641 224407409 111020931 -952290807 -351585540 479834398 820463230 479922168 492802252 -249125006 456069957 81354812 959801749 51995844 -171609313 830092352 -964046703 327434080 594150552 852794220 327286881 -811955817 432918314 -653574528 -486033168 -67248249 -65839338 -234459152 31231612 774839942 407737673 -388286827 526201415 20999335 -924744331 -405418645 -610275308 -680640139 -471956930 -863566761 -809329209 -63763498 -927855573 -576829982 -141047434 561499724 -862388737 -585184626 -505864883 -821107815 159276676 83222889 846500284 -514408860 -741816494 -841668660 -231867320 -211848991 998857275 726714249 -993119135 430252541 450929549 -790841019 -673866536 189071680 -399610540 -198961133 -77923445 279974583 3989696 -306592279 -307284785 -254338250 -26886039 225412384 587070749 447144161 87806592 884838057 -117820819 -71217975 638074771 730520934 605408353 132471548 -789299899 -183533797 687597337 632653783 -906675671 344506974 391567019 248183459 467665179 -924544293 -689023349 -243088540 -914398028 966171344 83179710 710460897 -351272158 290495539 400311929 318614032 -977908406 -268603195 273756317 750603639 -330532042 -961438918 915915938 -460914005 963742170 221725853 -561392487 914218775 477091747 463633319 529004450 818021844 -889300357 -525356364 965092149 -793530741 -548894247 -956271651 605378146 -600574951 -996648092 59097840 358197481 -29998838 -310065137 -967805951 -694519112 -733405313 -521456082 144266909 44287030 476046522 -303324647 -104865491 -30125383 843093251 -304947118 999649785 -126411301 530078138 610730353 -213669650 831171641 -569828828 -260310818 948997199 265996706 199579167 -921367940 -353926453 882444791 -688405027 -744559982 -547624622 -656653949 -966062819 343497624 646777138 996255196 240909126 -322249456 489606801 333993415 564569617 -31814017 552863966 -705874490 -549499837 465242120 329187433 -474756283 949445272 -910433626 -910912753 -328638371 49499324 746146309 144648718 -400857998 532451838 -125416368 463147223 -271440447 -312610387 -85274687 -87366095 -74292743 -88156262 -192007358 -391037184 157748503 800268891 41743688 -663964187 522423277 792229569 -146906266 -511589226 600772921 582464081 -49249107 446843382 720239449 -293094381 526941108 322769123 113122078 47208440 -684734792 747154109 647452088 54953704 -377872420 -850970763 185021393 -99967640 -383131759 -209693363 718606263 235461349 -902503664 197661233 -469388111 -566236257 -954590972 -503490382 -716152462 -706912459 918025798 962065051 610549675 -673792567 339096768 549837618 -620985669 472286671 540630643 -963036403 288522114 129956983 138200723 923010957 -371183789 926505653 114799311 24371196 -480595960 434536681 -134441297 -516922817 -898403537 -674301136 -351444201 927453496 682094177 -133297176 -495352134 869449395 -235555562 -311523898 -485799127 179196230 791453409 452435444 200166574 220742394 -892416506 -663081382 -236318699 -620377793 493579124 -636072023 -129018557 -506685941 688255657 543706005 110790999 -513226520 49021541 -356306623 319619163 482320165 378490352 419237305 356372647 157791532 -717154043 391047832 -325662254 859491060 114903130 965399754 926916950 847388045 564768973 764169580 499191487 222228268 810234256 244759641 502968993 851636124 262552242 370477605 472384440 -10801646 -888540263 666582774 510177807 -821899926 -452436746 -33531811 98808473 171589467 -167690478 -676538858 219076750 -940444744 -36166194 -651059337 753595123 -773527497 -561782220 808095235 -558892863 -313707954 489049017 260951835 -103313995 -287046346 827104018 212496084 315541939 125097490 493115567 -906179368 -136093656 636909784 333072923 712928077 -696656029 -961407588 439276190 203957519 696119331 -959755369 -966490132 -455927679 904239318 684525297 921113816 -287728083 -936368654 -418538051 -533862417 -982464377 730372116 -835188493 -601382354 134039732 927175263 -721234771 -997570035 -665037429 860886779 -801068610 -867234521 -962715989 105623229 701492496 -683464458 -498597979 382280052 -444484820 -956537551 -78547881 303966419 -249222695 771004863 -307052456 -72800266 659864027 671963734 -610706545 435438300 745832885 105207262 899937129 829683322 501995608 -924754157 -758996063 -890280095 -532114250 -737001831 371123034 260118230 192222468 -630394564 -134301660 191937548 241083829 427069686 570195489 -691659939 334821694 428316744 633662914 -591600212 257441495 -556517679 772305248 -3269594 727490953 135048369 944656722 -627340569 -823724994 -137823321 -745012196 695168244 102970692 447134535 701055124 597987011 -982206239 381974309 -542635166 -159331924 194363443 318639796 397822825 129141038 964154230 -804131905 545606097 555426864 -707695094 229628660 -268046145 -993081021 -907498142 711189698 -768603119 -598407442 321888471 19045633 365694924 772064440 -636826461 -498996478 659636317 625098997 -168687568 -470885126 -157260832 288084005 -134250654 -625324505 -57744836 841823746 -198601023 811235925 340478214 -871357882 -748343727 732463781 670133428 -133220309 -207962851 -775927350 -395815502 -912116457 -982599933 570377996 206431295 -419393422 151930137 -728348824 -287139869 959776408 -932259141 414899925 464524432 -779235021 877620129 -697000107 916837379 438611450 -351834951 664963025 870716809 34421434 41003761 561121081 -216903582 524982676 -318380409 -621708010 -493445886 799329223 269349728 342511886 319782277 635551055 715074910 -603657579 584470077 20109887 988172123 486119848 -916696624 -804682021 3223188 -339382153 367226552 -274115704 -299400129 -516284118 97364633 -276543062 -710559715 300212880 260090482 583430733 -77268536 -979697052 513288134 539025286 -167621320 536757323 777947457 959758064 885636118 72704854 875770783 -882794370 460081785 587368063 -989269075 374635444 -94868933 994925772 -956428558 -803492014 -796623114 768256822 -116840605 69687194 -809094708 -515414694 -424176380 180133415 -529891021 -370569333 -969026995 -430086678 -174307072 -667347053 559531821 -66675143 -39999164 445313575 -599359775 -539026201 -109255721 -320498953 74341990 834599290 141372315 706947474 815080616 -179241576 9467843 315338835 -242876879 -649256088 179121847 -64813235 -321209827 878827838 -98558142 -775037786 -411074770 -970740059 -439938996 -383237657 -899614340 -648513173 -253411427 -211766851 381691453 125059494 -274118717 650012176 -753462407 -34561290 -44954860 -506807495 121509830 949663051 719936355 105896537 102902846 -866112384 -59749168 377893191 713706050 -841009158 -490987860 -242053249 49601540 657614559 -288804276 -591057835 -362435906 -758189163 -709175883 -524504852 142066896 -98666789 985450328 558523273 195436319 -448952247 -437601506 -712813264 865140890 939936979 591624793 698567249 754639371 -693400104 991961452 575092965 788532801 -638408565 702363187 -391326443 -194257297 446818706 -36064265 -492700959 536316020 -580703924 357772851 -540180529 -855587686 830227359 850702022 -806835321 567940524 235312605 255421911 390384042 -758852654 -436128845 733737940 -748912506 -221481942 -285937181 180070997 729988345 -707798624 -987491810 -837374250 -918246773 924646993 231137490 -402372824 874612574 581642564 251386829 -16210270 -439106542 -21197910 678317437 -622618853 -251660848 127997875 793486275 496966740 83469923 416918285 517874144 -664493051 -890649472 405350138 -831291508 -231674750 -765684186 606121996 -808638683 -409506806 712584992 554561408 -901716732 818455212 -268268120 277205045 -385091092 983261399 -555573466 246665543 742892188 655338620 310480661 -591550388 848119428 306351842 -827334137 -494105302 -825647212 -457919226 -913310017 -540464748 -853455569 -675648340 500426620 158123551 406742078 -209049594 -840039083 916057861 486911586 -129543680 413140927 102263413 -961844671 -205456852 158595732 462061313 599758643 994300366 419104020 -652331821 185264951 -895888006 -867050897 304981159 387144803 429941159 -677316927 -959011980 41921173 -516934080 -861046308 416151738 995638922 69637977 266624475 988394692 -545838537 449168487 515095132 874823994 -428720366 928169177 -584157449 -618648387 164407106 752465750 -208577999 -660995525 -290572353 363694582 815619590 237126755 491846164 -341684203 35100170 -740934652 -762276524 423712649 71977002 -280257243 18492815 14088890 -992981787 -801651983 135480048 -151965735 -564516226 370530292 -717734931 -377239089 151581395 225412093 -728597996 -474358738 125392209 842397276 -576083063 64088821 -731219321 -681005977 464259764 284034715 777189035 494608985 -905358017 924979033 -569849251 697383134 -97260969 -677665823 -77812545 -424972379 677487036 -870472292 97707502 285985361 991299992 -398831575 759643502 -756645742 -701853474 393550030 827553761 75379187 -926226434 247313679 813244080 -412468652 -807520477 -479441567 522650065 415396073 -769966586 493414647 -285684680 608212315 615012205 853836985 171652316 -906102204 -557881517 698218084 -292572552 -212733354 664757354 656758638 408033463 414919336 -437133080 -192959108 -446839522 -115244174 -977237384 -125836164 999702790 378195387 -612576242 776254707 412091812 -586307952 -808097379 421843653 -312351721 -62616608 527035628 225640099 998091649 -131652361 451177859 204571665 341137557 -138072641 -981774829 -560992370 868182699 -920361211 -748594061 -275757689 667941809 -487137517 -539098190 -779610553 756891762 820773257 -473338515 -384990554 181618696 879104026 170164746 -255147743 772995530 313803192 728188835 -151873091 -300478641 -922890923 974104267 -507777794 825533298 26135902 -990371927 949284107 778263091 -315188111 -487402201 443200327 -801669203 -534746430 -391423312 -297681586 733559807 -335449861 864216076 -797312046 -538280614 -769767768 -32622293 531653546 -632440034 7930097 -515478735 148122592 118701553 -39988613 626460091 -538190482 451582450 825586757 109623987 395704212 316256863 -105295084 -6430253 257727671 -157185828 -942387717 956588542 -292238919 823424866 47815420 -499739075 584085604 663084743 -312863799 656099412 651855615 -541985819 434261155 -833405181 -763562646 854587236 388914640 -403458840 72594492 721118366 -127694056 778827910 -140995387 -75490136 109052421 -448664220 -756896821 414251342 253443926 957898023 98973798 518910570 716412476 932456517 -777387567 304836167 512606947 -295173920 89894277 125145408 -249943291 889984241 790425376 -441799433 -826518120 -11669231 -952069024 -182769362 286059432 515868132 891621198 470343749 -90065857 38853154 -959299465 829169024 944673603 567960405 -682661419 -520747821 137895623 -388358451 -716887777 894275642 570615327 57793490 981286721 40635074 -468936901 517121063 -254076501 -277817834 114633649 -182785215 943959906 407413550 -835061625 233030474 439672543 -731233244 -705318755 -123396030 975081350 693471527 -683809862 975170368 900870337 -258622840 907355185 -353974313 -120794951 575693298 -554590929 -91583084 208250033 402797751 189432740 -151152733 449740030 -99151547 -98866228 579939472 -629064025 -72932937 -955044519 -157130709 -789424760 -661348289 -641732218 -384832905 139756642 -863364069 -605501672 -294419659 629124116 -621722559 177916676 884792590 820960163 -107787680 316934515 -614857392 685473143 389444052 631509789 871488947 111661750 -531087991 -129231533 -288583250 281825926 767147431 -183142583 -955991700 -351094499 -666120445 -762558236 768724074 -528955730 521277047 115478921 -794573724 113416723 -721287819 -413415900 -32296629 -170937173 -359730614 134994438 433928783 -416671507 89600039 -64618454 -523354161 824190651 756845372 -937837952 985405764 645034003 419958862 -623749980 826366648 -167503822 -346350869 764101284 673094040 -936012729 987138946 423064491 47214127 -875305417 939032315 -598909654 605462287 -487491602 662555403 842624650 825134621 371543693 -300460315 -86833979 -155552347 575972876 -203969445 966218061 -839251137 -212048291 -699240686 133015367 353491016 -660988055 -300645938 -336210655 370687434 555256073 -314715607 -961511210 582114314 36145775 -427541049 -375478723 -599623345 946161432 54848932 -760259844 557064470 -549257596 -402696059 804265043 711931925 351652280 -598746568 705787176 -478615059 249473657 428712918 -329880027 303525119 -509914824 -197580835 644858842 713886249 337495835 514441343 67533254 126433483 -620091030 313213232 451701208 -222438378 -345211708 -13727921 -676350774 -750381140 -268462489 -624643826 -560021051 -580166717 865974492 730774999 -774807113 782036000 681868514 -943265197 478626592 -284553811 382373921 -668817266 205781467 515173717 -711066792 829082227 930837967 413771413 714658460 595873571 -307462608 654628973 -802379076 -869715726 559739138 349849683 -562862517 866037424 744505387 -191528387 525838083 392848946 149104604 -769794696 293472240 -375706470 724222185 740086062 -640912264 223428053 973864599 833264375 584195303 21988392 830065589 -473492984 -154420072 -764778852 214735208 155181691 -335146923 573879684 447859869 -803429495 543592613 -521765132 -278734220 -675633145 726486339 -504110954 619009356 -31360459 -34421028 957863900 -104169971 810824771 -9999770 -236543155 -779460591 334012137 -356800767 472154544 -263825356 13713424 -404494808 -687418522 -225287071 -945553802 -542817340 -262395997 267365090 644727149 341040247 -93446864 -461176451 225723009 -903222410 -77920711 984713108 -998720062 -802435939 345961979 125449087 607596675 -316093027 -562577653 -568817233 486711376 798903360
