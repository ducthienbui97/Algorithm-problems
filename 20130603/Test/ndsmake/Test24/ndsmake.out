1228670351136
